jaxtyping/_import_hook.py

```diff
@@ -76,8 +76,11 @@ def _optimized_cache_from_source(typechecker_hash, /, path, debug_override=None)
     # Version 5: Added support for string-based `typechecker` argument.
     # Version 6: optimization tag now depends on `typechecker` argument, so that
     #    changing the typechecker will hit a different cache.
+    # Version 7: Now allowing to pass functions as a typechecker, using md5 hash of the
+    #    source code as the hash.
```

Comment:
I think this statement is no longer true. :)

Reply:
Yep, nice catch!